arelle/ModelRenderingObject.py

```diff
@@ -22,6 +22,27 @@
 
 EMPTY_SET = set()
 
+class LayoutMdlHdrCells:
```

Comment:
Suggest using `@dataclass` for all of these. Defines a bunch of the boilerplate stuff for you.